### PR TITLE
Updated monthly statistics

### PR DESCRIPTION
**Pull request by zapbot**
Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

File: site/data/charts/headline.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-month: February 2023
-zap_runs: 1,308,922
-zap_ascans: 807,194
-zap_alerts: 864,973,499
-zap_attacks: 1,830,861,939
+month: March 2023
+zap_runs: 1,517,688
+zap_ascans: 5,194,372
```
Comment (thc202) [inline site/data/charts/headline.yaml:3]: Big jump, worth double checking @psiinon ?

Comment (psiinon) [reply site/data/charts/headline.yaml:3]: Yeah, will do...

**Comment by psiinon** [comment]
Doesnt look obviously manipulated, and we should always take such stats with a pinch of salt...

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you for double checking!

**Pull request merged**